tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -334,8 +334,10 @@ def _run(self):
                     out_lines = stdout.split('\n')
                     connected = True
                     for index in range(4, len(out_lines) - 2):
-                        if out_lines[index].split('\t')[2].strip() != 'Connected':
-                            connected = connected and False
+                        node_status_det = out_lines[index].split('\t')
+                        if len(node_status_det) > 2:
+                            if node_status_det[2].strip() != 'Connected':
+                                connected = connected and False
```

Comment:
I want to AND the previous value with `False` if current node is not `Connected`.
So at the end if any of the nodes is not `Connected` we will get a False value.